### PR TITLE
(DOCSP-17097)(DOCSP-17262): improved connection example and added arb…

### PR DESCRIPTION
**Pull request by jwilliams-mongo**
…iters

https://jira.mongodb.org/browse/DOCSP-17262
https://jira.mongodb.org/browse/DOCSP-17097

### All Submissions:

* [x] Have you opened an Issue before filing this PR?
* [x] Have you signed our [CLA](https://www.mongodb.com/legal/contributor-agreement)?
* [x] Have you signed all of your commits?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [x] Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).

### CODE REVIEW

File: docs/deploy-configure.md [hunk 1]
```diff
@@ -121,6 +134,65 @@ To scale a replica set:
...
    members that you remove from the replica set.
 
+## Add Arbiters to a Replica Set
+
+You can add [arbiters](https://docs.mongodb.com/manual/core/replica-set-arbiter/) to your replica set by adding the `spec.arbiters` field to your MongoDB resource definition. 
+
+The value of the `spec.arbiters` field must be:
+
+- a positive integer, and
+- less than the value of the `spec.members` field. At least one replica set member must not be an arbiter.
```
Comment (corryroot) [inline docs/deploy-configure.md:144]: **Suggestion:** This last sentence looks a little awkward as a continued thought to the requirement. I suggest making the sentence  its own line or maybe a note.

```suggestion
- less than the value of the `spec.members` field.
  At least one replica set member must not be an arbiter.
```

File: docs/deploy-configure.md [hunk 2]
```diff
@@ -121,6 +134,65 @@ To scale a replica set:
...
+
+Consider the following example MongoDB resource definition:
```
Comment (corryroot) [inline docs/deploy-configure.md:146]:
```suggestion
Consider the following MongoDB resource definition example:
```

File: docs/deploy-configure.md [hunk 3]
```diff
@@ -69,13 +70,25 @@ To deploy your first replica set:
...
-5. Use one of the connection strings returned in the previous step to connect to the replica set:
+5. Connect to one of your application's pods in the Kubernetes cluster:
+
+   **NOTE**: You can access your replica set only from a pod in the same Kubernetes cluster. You can't access your replica set from outside of the Kubernetes cluster.
+
+   ```
+   kubectl -n <my-namespace> exec --stdin --tty <your-application-pod> -- /bin/bash
+   ```
+
+   When you connect to your application pod, you see a shell prompt for your application's container:
```
Comment (corryroot) [inline docs/deploy-configure.md:81]:
```suggestion
   When you connect to your application pod, a shell prompt appears for your application's container:
```

File: docs/deploy-configure.md [hunk 4]
```diff
@@ -121,6 +134,65 @@ To scale a replica set:
    might take several minutes to remove the StatefulSet replicas for the
    members that you remove from the replica set.
 
+## Add Arbiters to a Replica Set
+
+You can add [arbiters](https://docs.mongodb.com/manual/core/replica-set-arbiter/) to your replica set by adding the `spec.arbiters` field to your MongoDB resource definition.
```
Comment (corryroot) [inline docs/deploy-configure.md:139]:
```suggestion
To add [arbiters](https://docs.mongodb.com/manual/core/replica-set-arbiter/) to your replica set, add the `spec.arbiters` field to your MongoDB resource definition. 
```

**Review by corryroot — APPROVED** [review]
LGTM! I added some minor tweaks and suggestions that you can add or ignore.

File: docs/deploy-configure.md (hunk 1 above)
Comment (jwilliams-mongo) [reply docs/deploy-configure.md]: I had this as a note in an earlier draft. Moved it back out to a note.

**Review by priyolahiri — APPROVED** [review]
LGTM

**Pull request merged**